src/jc21/Item.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace jc21;
+
+class Item
+{
```

Comment:
Could this class have something useful? maybe jsonSerialize?

Reply:
The class is just intended to be a parent class so that you can put either the Movie or TV class into the same ItemCollection, otherwise you'd have to have 2 different collection classes.  I had intended on adding Photos and Music as options as well, but it appears that Plex treats those differently.
If you were to put jsonSerialize in the Item class it would only have access to properties with the Item class.  Now that I think about it, it may make more sense to make Item an interface so you could force compliance...I think this would still allow either class "implementing" the interface to be added to the collection.  I'll have to try that out.